beta/src/components/MDX/APIAnatomy.tsx

```diff
@@ -101,9 +101,6 @@ export function AnatomyStep({
   const color = colors[stepNumber - 1];
   return (
     <div
-      style={{
-        borderLeftColor: color.hex,
```

Comment:
?

Reply:
We have class to add border color and have borderleft color in the style for the same
https://github.com/reactjs/reactjs.org/blob/d7cbb1bab155f28519e86ddfff453896cb359568/beta/src/components/MDX/APIAnatomy.tsx#L109